GNUmakefile

```diff
@@ -11,6 +11,12 @@ ACCTEST_COUNT       ?= 1
 GOFMT_FILES         ?=$$(find . -name '*.go' |grep -v vendor)
 PKG_NAME            =equinix
 
+CRI=docker
```

Comment:
Rather than add a dependency on `docker` which devs may not have, perhaps we could just use `go run`:

```
 go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.59.1 run
```